src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/Create/cart.html.twig

```diff
@@ -97,10 +97,12 @@
               </div>
               <div class="col-2">
                 <input name="product_quantity" id="quantity-input" type="number" min="1" value="1" class="form-control">
+                {% if stockManagementEnabled %}
                 <small class="form-text">
```

Comment:
Could you please fix the indentation?

Reply:
Done in the commit https://github.com/PrestaShop/PrestaShop/pull/23811/commits/18c98cabfd2ebccc51a1e2e764a9e85598eec1ef